doc/getting_started/installation_linux.rst

```diff
@@ -66,6 +66,11 @@ Install the required packages in a Fedora host system with:
 	 libstdc++-static python3-ply ncurses-devel \
 	 python-yaml python2 dfu-util
 
+.. note::
```

Comment:
+1 on the approach. We probably could document long-standing issues with boards supported by Zephyr (well, I tried to do that, so it must be good ;-) ), but can't turn documentation into a bugtracker for 3rd-party apps/systems. Perhaps, that can be addressed by a separate "Known Issues" doc for each release, and it would consist of links to Jira issues, so people could see any updates in status beyond the time when the doc was written.